### DISCUSSION TITLE
PrimeNG table filtering and sorting on each column- filter is not opening with keyboard navigation on v17.1.0

### DISCUSSION THREAD

**Discussion by vanamaa** [primefaces/community#916] [PrimeNG] [answered]
HI Team,

I implemented filtering and sorting (both) on each column on table. When I am trying to use key board navigation (using tab and space bar), sorting is working but not able to open filter.

using angular 17.0.7, primeng 17.1.0 
but same working on 13 version

this is not even working on site https://primeng.org/table 

![Capture](https://github.com/primefaces/community/assets/26254651/7aea9659-e087-4c39-9e7d-83e4f4a1b0eb)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks a lot for reporting the issue!
I opened an issue now and the issue will be addressed in the upcoming release of PrimeNG (17.3.3) 
https://github.com/primefaces/primeng/issues/14545

**Reply by vanamaa** [reply]
Thanks for your quick reply. 
We are planning to show a POC , I would like to know below 

May I know when is next release(17.3.3) scheduled ? 
Do we have any work around(temporary way to fix) for this issue?

**Reply by cetincakiroglu** [reply]
We're planning to release tomorrow
